model/otel/metrics.yaml

```diff
@@ -1,27 +1,27 @@
 groups:
-  - id: metric.otel.sdk.span.live.count
+  - id: metric.otel.sdk.live_spans
     type: metric
-    metric_name: otel.sdk.span.live.count
+    metric_name: otel.sdk.live_spans
```

Comment:
same here, wouldn't `otel.sdk.span.live` be more consistent?

Reply:
related to #1999

I think I like your suggestion

* `otel.sdk.span.ended`
* `otel.sdk.span.live` (or `otel.sdk.span.active`)

especially if we also like this pattern for `http.server.active_requests` and rename that to `http.server.request.active`

Reply:
Fixed in https://github.com/open-telemetry/semantic-conventions/pull/2042/commits/6af9056f429677cda826f9162e0bf793675f95ff . I stayed with `live` because I personally find `active` confusing in the context of spans because to me that is ambigous with a span being part of the current context of a thread, which is independent of a span being ended or not.